pkg/webhook/common.go

```diff
@@ -93,6 +94,11 @@ func (h *webhookHandler) tracingLevel(ctx context.Context, req admission.Request
 	return traceEnabled, dump
 }
 
-func (h *webhookHandler) skipExcludedNamespace(namespace string) bool {
-	return h.processExcluder.IsNamespaceExcluded(process.Webhook, namespace)
+func (h *webhookHandler) skipExcludedNamespace(obj runtime.Object) (bool, error) {
```

Comment:
placing the conversion logic here will avoid duplication later when we integrate mutation

Reply:
moved inside skipExcludedNamespace